front/app/modules/commercial/analytics/admin/hooks/useRegistrations/parse.ts

```diff
@@ -31,13 +31,11 @@ const parseRow = (date: Moment, row?: TimeSeriesResponseRow): TimeSeriesRow => {
   };
 };
 
-const getDate = dateGetter<TimeSeriesResponseRow>(
-  'dimension_date_registration'
-);
+const getDate = dateGetter('dimension_date_registration');
 const _parseTimeSeries = timeSeriesParser(getDate, parseRow);
 
 export const parseTimeSeries = (
-  responseTimeSeries: TimeSeriesResponseRow[],
+  responseTimeSeries: Response['data'][0],
```

Comment:
why make this less specific?

Reply:
Because the time series here can also be an empty array. To be correct we could also do `TimeSeriesResponse || []`, or `TimeSeriesResponseRow[] || []`. But if that changes (because we decide to never return empty arrays) we now only have to change that in one place. Also now this is consistent across different hooks

Reply:
ok cool